### PR TITLE
feat: pipe logging to single process

### PR DESCRIPTION
**Pull request by TomMcL**
### Description

Piping all log outputs from process management process to the main process to allow output to files etc

### Testing

Ran locally, output saved correctly

### Breaking Changes

Should for the most part act as before. Saved log files may have a couple more lines in

### CODE REVIEW

File: tests/integration/utils/fixtures.py [hunk 1]
```diff
@@ -144,7 +144,7 @@ def vega_service():
         retain_log_files=True,
         transactions_per_block=1,
         listen_for_high_volume_stream_updates=False,
-        use_full_vega_wallet=True,
+        use_full_vega_wallet=False,
```
Comment (cdummett) [inline tests/integration/utils/fixtures.py:147]: Is this intentional? Think we switched from `False` -> `True` yesterday and wonder if this is a conflict.

File: tests/integration/utils/fixtures.py [hunk 2]
```diff
@@ -144,7 +144,7 @@ def vega_service():
         retain_log_files=True,
         transactions_per_block=1,
         listen_for_high_volume_stream_updates=False,
-        use_full_vega_wallet=True,
```
Comment (cdummett) [inline tests/integration/utils/fixtures.py:147]: Is this intentional? Think we switched from `False` -> `True` yesterday and wonder if this is a conflict.

File: tests/integration/utils/fixtures.py [hunk 3]
```diff
@@ -164,7 +164,7 @@ def vega_service_with_high_volume():
         retain_log_files=True,
         transactions_per_block=1,
         listen_for_high_volume_stream_updates=True,
-        use_full_vega_wallet=True,
+        use_full_vega_wallet=False,
```
Comment (cdummett) [inline tests/integration/utils/fixtures.py:167]: ditto

File: tests/integration/utils/fixtures.py (hunk 2 above)
Comment (TomMcL) [reply tests/integration/utils/fixtures.py:147]: No the switch to `True` was the non-deliberate part 😁

**Review by cdummett — APPROVED** [review]

**Pull request merged**